WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -846,7 +847,12 @@ private void handleNewStoryAction(PagePostCreationSourcesDetail source) {
         SiteModel site = getSelectedSite();
         if (site != null) {
             // TODO: evaluate to include the QuickStart logic like in the handleNewPostAction
-            ActivityLauncher.addNewStoryForResult(this, site, source);
+            ActivityLauncher.showPhotoPickerForResult(
```

Comment:
now instead of showing the Story Composer, we launch the media picker when the user taps on the FAB buttton -> create story